### PR TITLE
feat(unlock-protocol-com): Add blog post for redesigned checkout

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by searchableguy** [comment]
Opened PR for adding new configuration option here: https://github.com/unlock-protocol/docs/pull/60

File: unlock-protocol-com/blog/redesigning-our-checkout.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: The brand new checkout experience
+authorName: searchableguy
+publishDate: July 5, 2022
```
Comment (julien51) [inline unlock-protocol-com/blog/redesigning-our-checkout.md:4]:
```suggestion
publishDate: July 6, 2022
```

File: unlock-protocol-com/blog/redesigning-our-checkout.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+### Choosing Quantity
+
+You will be able to select quantity and payment method you want to use earlier in the flow.
+
+![quantity selection on checkout](/images/blog/redesigned-checkout/new-checkout-quantity.jpg)
+
+### Recipient and metadata collection
+
+Recipients field will be populated based on the quantity. You will be able to add recipient addresses and fill out information. We've added more validation checks to ensure user cannot buy memberships than allowed by the locker owner early in the flow.
```
Comment (julien51) [inline unlock-protocol-com/blog/redesigning-our-checkout.md:31]:
```suggestion
Recipients field will be populated based on the quantity. You will be able to add recipient addresses and fill out information. We've added more validation checks to ensure user cannot buy memberships than allowed by the lock owner early in the flow.
```

File: unlock-protocol-com/blog/redesigning-our-checkout.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+### Payment using card
+
+We've made it easier to pick and change card using the checkout. We're planning to introduce more traditional payment options soon.
+
+![card selection on checkout](/images/blog/redesigned-checkout/new-checkout-add-card.png)
+
+### Confirmation
+
+Our new confirmation page will show you how many memberships you are buying, what the total amount is, how many times a membership will recurr if it's a recurring membership and more.
```
Comment (julien51) [inline unlock-protocol-com/blog/redesigning-our-checkout.md:43]:
```suggestion
Our new confirmation page will show you how many memberships you are buying, what the total amount is, how many times a membership will be renewed if it's a recurring membership and more.
```

**Review by julien51 — APPROVED** [review]
Looking good! 
Let's make sure @ccarfi gets his eyes on it too and publish once we have the production version live!

File: unlock-protocol-com/blog/redesigning-our-checkout.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+### Choosing Quantity
+
+You will be able to select quantity and payment method you want to use earlier in the flow.
+
+![quantity selection on checkout](/images/blog/redesigned-checkout/new-checkout-quantity.jpg)
+
+### Recipient and metadata collection
+
+Recipients field will be populated based on the quantity. You will be able to add recipient addresses and fill out information. We've added more validation checks to ensure user cannot buy memberships than allowed by the lock owner early in the flow.
```
Comment (searchableguy) [inline unlock-protocol-com/blog/redesigning-our-checkout.md:31]:
```suggestion
Recipients field will be populated based on the quantity. You will be able to add recipient addresses and fill out information. We've added more validation checks to ensure user cannot buy more memberships than allowed by the lock owner early in the flow.
```

**Comment by ccarfi** [comment]
A few typos here and there. I'm going to go in and edit directly. Will note back here and do final review/approval when updated.

**Pull request merged**